### PR TITLE
fix: 🐛 Move Optimonk banner down by setting its z-index

### PR DESCRIPTION
**Pull request by ashleytowner**
# Pull Request Overview
The Optimonk banner has been displaying over the cookie popup, because its z-index is really high. This PR reduces that to 99 (under the cookie popup)

This fixes [this ticket](https://www.notion.so/greenpeaceaustraliapacific/P4-Z-index-on-Cookies-policy-3b90a69a06f14b03a95f1220c8fb736c?pvs=4)

> [!WARNING]
> We still need to remove this code snippet which was added to google tag manager

|Before|After|
|:-:|:-:|
|image|image

### CODE REVIEW

File: style/global.scss [hunk 1]
```diff
@@ -131,3 +131,9 @@ mark[style="background-color:rgba(0, 0, 0, 0)"] {
 .with-mobile-tabs ~ :not(.page-header) + .page-content {
   padding-top: 0;
 }
+
+// This is an optimonk thing, their z-index is ridiculously high and we can't
+// display anything above it, so we are setting it lower here.
```
Comment (techieshark) [inline style/global.scss:136]:
```suggestion
// display anything above it, so we are setting it lower here.
// We were prompted to do this so it sits below our cookie consent block.
// See https://github.com/greenpeace/planet4-child-theme-australiapacific/pull/77/
```

Comment (techieshark) [reply style/global.scss:136]: Just to connect the dots for the reader

**Review by techieshark — APPROVED** [review]
one nit, but lgtm

**Comment by ashleytowner** [comment]
@techieshark comitted your suggestion (just squashed it into the previous commit) 

Although, we can just search for the commit in the PRs tab to find all the PRs it is in.

image

**Comment by techieshark** [comment]
@ashleytowner I am aware of that, but also aware of how annoying it is to have to do that :)

**Comment by ashleytowner** [comment]
> @ashleytowner I am aware of that, but also aware of how annoying it is to have to do that :)

Fair

**Review by techieshark — APPROVED** [review]
lgtm

**Pull request merged**